packages/vkui/src/components/HorizontalCell/CellEllipsisText/EllipsisText.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { classNames } from '@vkontakte/vkjs';
+import { Typography, type TypographyProps } from '../../Typography/Typography';
+import styles from './EllipsisText.module.css';
+
+export interface EllipsisTextProps extends TypographyProps {
+  /**
+   * Параметр ширины компонента. Должен дублировать ширину контейнера
+   *  когда размер произвольный и не равен рекомендуемым 128px
+   *  для размера HorizontalCell size="l"
+   */
```

Comment:
```suggestion
  /**
   * Пользовательская маскимальная ширина.
   *
   * Используйте в случаях, когда у контейнера ширина зависит от размера контента,
   * другими словами, когда ширина не ограничена.
   */
```

тут опять же отвязываемся от `HorizontalCell`